test/service/test_manifest.py

```diff
@@ -1650,15 +1650,17 @@ class TestManifestExpiration(AzulUnitTestCase):
 
     def test_get_seconds_until_expire(self):
         """
-        Verify a header with valid Expiration and LastModified values returns the correct expiration value
+        Verify a header with valid Expiration and LastModified values returns
+        the correct expiration value.
         """
-        margin = ManifestService._date_diff_margin
-        for object_age, expect_error in [(0, False),
-                                         (margin - 1, False),
-                                         (margin, False),
-                                         (margin + 1, True)]:
+        for object_age, expect_error in [(1, False),
+                                         (0, False),
+                                         (-1, True)]:
             with self.subTest(object_age=object_age, expect_error=expect_error):
-                with mock.patch.object(manifest_service, 'datetime') as mock_datetime:
+                with mock.patch.object(manifest_service,
+                                       'datetime',
+                                       mock.Mock(wraps=datetime)
```

Comment:
Is the `wraps` needed? What for?

Reply:
`wraps` was needed so that a new `datetime` object could be created in `_get_seconds_until_expire`, however now with the `datetime.replace` approach `wraps` is no longer required and the change here will be reverted.